src/os/bluestore/BlockDevice.cc

```diff
@@ -19,7 +19,7 @@
 
 #include "BlockDevice.h"
 
-#if defined(HAVE_LIBAIO)
+#if defined(HAVE_LIBAIO) || defined(HAVE_POSIXAIO)
```

Comment:
this commit and 544b4f70f47d82ebc23726bc6b6eafcab61d42ff come from Alan Somers's branch at https://github.com/asomers/ceph/tree/bluestore_fbsd . i'd suggest 

1. squash 544b4f70f47d82ebc23726bc6b6eafcab61d42ff into this commit
1. keep Alan's "Signed-off-by: Alan Somers <asomers@gmail.com>" in this commit and remove yours. because i don't think there is enough creative work from you after reading through this commit and https://github.com/asomers/ceph/commit/3bf3cbcf19a4b117f1589d1f9cd3d246f8666dc0 .

Reply:
@tchaikov 
I don't claim originality for this work. Which is why I attribute the work to Alan's PR/branch. No problem with changing Signed-off.